AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -440,30 +421,18 @@ def update_task(self, task, new_lease_seconds):
         represents the number of seconds from now.
     """
     new_eta = current_time_ms() + datetime.timedelta(seconds=new_lease_seconds)
-
-    check_lease = ''
-    if hasattr(task, 'leaseTimestamp'):
-      check_lease = 'AND lease_expires = %(old_eta)s'
-
-    update_task = """
-      UPDATE pull_queue_tasks
-      SET lease_expires = %(new_eta)s
-      WHERE app = %(app)s AND queue = %(queue)s AND id = %(id)s
-      IF lease_expires > dateof(now())
-      {check_lease}
-    """.format(check_lease=check_lease)
     parameters = {
       'app': self.app,
       'queue': self.name,
       'id': task.id,
-      'new_eta': new_eta
+      'new_eta': new_eta,
+      'current_time': datetime.datetime.utcnow()
     }
-    if check_lease:
+    if hasattr(task, 'leaseTimestamp'):
```

Comment:
@cdonati , it seems that the same problem (processing of possible absence of an attribute) is solved in different way previously in this file. I wouldn't strongly recommend to use try/except, but it seems that try/except is more pythonic way: https://docs.python.org/2/glossary.html#term-eafp. Anyway it makes sense to keep style the same inside a file.